inc/spbc-tools.php

```diff
@@ -251,4 +251,24 @@ function spbc_wp_timezone_string(){
     $abs_mins = abs( $minutes * 60 );
     
     return sprintf( '%s%02d:%02d', $sign, $abs_hour, $abs_mins );
+}
+
+/**
+ * Checks if the string is ASCII
+ *
+ * @param $string
+ *
+ * @return bool
+ */
+function spbc_check_ascii($string)
+{
+    if( function_exists('mb_check_encoding') ){
+        if( mb_check_encoding($string, 'ASCII') ){
+            return true;
+        }
+    }elseif( ! preg_match('/[^\x00-\x7F]/', $string) ){
```

Comment:
Maybe this "not" operator is redundant here?

Reply:
This function was borrowed from WPDB library. I suppose this is correct.